Engine/Module/ModuleResourceManager.cpp

```diff
@@ -13,13 +43,23 @@ ModuleResourceManager::ModuleResourceManager()
 bool ModuleResourceManager::Init()
 {
 	APP_LOG_SECTION("************ Module Resource Manager Init ************");
-	texture_importer = std::make_unique<TextureImporter>();
+
+	animation_importer = std::make_unique<AnimationImporter>();
 	material_importer = std::make_unique<MaterialImporter>();
+	mesh_importer = std::make_unique<MeshImporter>();
 	model_importer = std::make_unique<ModelImporter>();
-	scene_manager = std::make_unique<SceneManager>();
 	prefab_importer = std::make_unique<PrefabImporter>();
+	skeleton_importer = std::make_unique<SkeletonImporter>();
+	skybox_importer = std::make_unique<SkyboxImporter>();
+	texture_importer = std::make_unique<TextureImporter>();
+
+	metafile_manager = std::make_unique<MetafileManager>();
+	scene_manager = std::make_unique<SceneManager>();
+
+	ImportAssetsInDirectory(*App->filesystem->resources_folder_path); // Import all assets in folder Resources. All metafiles in Resources are correct"
+
 	importing_thread = std::thread(&ModuleResourceManager::StartThread, this);
-	File("Resources");
+	//StartThread();
```

Comment:
why is this here and commented?¿

Reply:
For debugging purposes, it will be deleted at the future